data/database.sql

```diff
@@ -363,7 +363,7 @@ CREATE TABLE IF NOT EXISTS `site_scan_history` (
   CONSTRAINT `fk_site_scan_history_sites1`
     FOREIGN KEY (`sites_id`)
     REFERENCES `sites` (`id`)
-    ON DELETE NO ACTION
+    ON DELETE CASCADE
```

Comment:
My understanding of cascading updates is that it will update foreign key references if a primary key changes.  No where in the system will we change a primary key, so on update isn't necessary (but I guess it wouldn't hurt either).